lib/active_merchant/billing/gateways/nuvei.rb

```diff
@@ -152,11 +152,20 @@ def add_account_funding_transaction(post, payment, options = {})
         return unless options[:is_aft]
 
         recipient_details = {
+          firstName: options[:aft_recipient_first_name],
+          lastName: options[:aft_recipient_last_name]
+        }.compact
+
+        address_details = {
           firstName: payment.first_name,
           lastName: payment.last_name,
-          country: options.dig(:billing_address, :country)
```

Comment:
Maybe you could assing billing_address to a variable to avoid accessing the option hash each time you require a field.